includes/class-wc-taxjar-connection.php

```diff
@@ -2,86 +2,212 @@
 /**
  * TaxJar Connection
  *
```

Comment:
Most of the changes to this class were made to make the code easier to follow. The $integration parameter was also removed from the constructor to untie the two classes.